source/chapters/setup.rst

```diff
@@ -85,86 +137,111 @@ Laptop and an External USB Soundcard
+**Using two soundcards (built-in + external)**
 
 #. Open :menuselection:`Preferences --> Sound Hardware`
 #. Select the :guilabel:`Output` tab
-#. From the :guilabel:`Master` drop-down menus, select the external soundcard,
+#. From the :guilabel:`Master` drop-down menu, select the external soundcard,
    then :guilabel:`Channels 1-2`
-#. From the :guilabel:`Headphones` drop-down menus, select the external
-   soundcard, then :guilabel:`Channels 3-4`
+#. From the :guilabel:`Headphones` drop-down menu, select the built-in
+   soundcard, then :guilabel:`Channels 1-2`
 #. Click :guilabel:`Apply` to save the changes.
 
-**Using two soundcards (built-in + external)**
+**Using an external multi-channel soundcard**
 
 #. Open :menuselection:`Preferences --> Sound Hardware`
 #. Select the :guilabel:`Output` tab
-#. From the :guilabel:`Master` drop-down menus, select the external soundcard,
+#. From the :guilabel:`Master` drop-down menu, select the external soundcard,
    then :guilabel:`Channels 1-2`
-#. From the :guilabel:`Headphones` drop-down menus, select the built-in
-   soundcard, then :guilabel:`Channels 1-2`
+#. From the :guilabel:`Headphones` drop-down menu, select the external
+   soundcard, then :guilabel:`Channels 3-4`
 #. Click :guilabel:`Apply` to save the changes.
 
+.. hint:: If you have technical difficulties with your external sound card, a
+          :ref:`splitter cable<setup-laptop-with-splitter>` can be used as a
+          portable, inexpensive backup.
+
+.. note:: Soundcards designed and marketed for recording often have a knob that 
+          allows you to mix its input directly with its output without the 
+          added :term:`latency` of sending the input into the computer. 
+          This function can be helpful when recording, but is not as useful
+          while DJing. To get the maximum output volume from these soundcards,
+          you may need to set this knob to only send output from the computer
+          and none from the input(s).
+
 .. _setup-controller-and-external-card:
 
-Laptop, MIDI Controller, and External USB Soundcard
----------------------------------------------------
+Laptop and MIDI or HID Controller
+=================================
 
-Mixxx can work with any :term:`MIDI controller` that has drivers for your
-:term:`OS <operating system>`, as long as there is a MIDI mapping file to tell
-Mixxx how to understand it. Mixxx comes bundled with a number of MIDI mapping
-presets. To see the full list, see :ref:`control-midi`.
+Most DJs prefer the tactile and intuitive control provided by DJ MIDI 
+controllers over only a keyboard and mouse. Mixxx can work with any :term:`MIDI 
+controller` as long as there is a MIDI mapping file to tell Mixxx how to 
+understand the controller's signals. Mixxx comes bundled with a number of MIDI
+mapping presets. To see the full list of included mappings, see
+:ref:`control-midi`. For controllers that Mixxx does not yet support, anyone can 
```

Comment:
"for controllers that mixxx does not yet support, check the forums to see if someone has created a mapping [link], or you can do the mapping yourself [link"